src/cljs/ui/analysis_results/page.cljs

```diff
@@ -42,11 +44,13 @@
      [discrete-rates analysis]
      nil)])
 
-(defn results [{:keys [bayes-factors] :as analysis}]
+(defn results [{:keys [bayes-factors custom-map] :as analysis}]
   (let [config @(re-frame/subscribe [::subs/config])
         viewer-host (:analysis-viewer-url config)
         {:keys [viewer-url-params]} (:analysis analysis)
-        viewer-url (str viewer-host "/" viewer-url-params)]
+        [_ custom-map-url] (when custom-map (re-find #".+://.+?/.+?/(.+)" (:file-url custom-map)))
```

Comment:
what's the deal here with the regex? Can you leave a comment or refactor to utility function?

Reply:
uhm,.. I see no changes :)